python/lsst/rapid/analysis/quickLook.py

```diff
@@ -0,0 +1,281 @@
+        isrConfig.doLinearize = False
+        isrConfig.doCrosstalk = False
+        isrConfig.doBrighterFatter = False
+        isrConfig.usePtcGains = False
+
+        if bias:
+            isrConfig.doBias = True
+            self.log.info("Running with bias correction")
+
+        if dark:
+            isrConfig.doDark = True
+            self.log.info("Running with dark correction")
+
+        if flat:
+            isrConfig.doFlat = True
+            self.log.info("Running with flat correction")
+
+        # TODO: deal with fringes here
+        if defects:
+            isrConfig.doDefect = True
+            self.log.info("Running with defect correction")
+
+        if linearizer:
+            isrConfig.doLinearize = True
+            self.log.info("Running with linearity correction")
+
+        if crosstalk:
+            isrConfig.doCrosstalk = True
+            self.log.info("Running with crosstalk correction")
+
+        if bfKernel:
+            isrConfig.doBrighterFatter = True
+            self.log.info("Running with brighter-fatter correction")
+
+        if ptc:
+            isrConfig.usePtcGains = True
+            self.log.info("Running with ptc correction")
+
+        isrConfig.doWrite = False
+        isrTask = IsrTask(config=isrConfig)
+        result = isrTask.run(ccdExposure,
+                             camera=camera,
+                             bias=bias,
+                             dark=dark,
+                             flat=flat,
+                             #  fringes=pipeBase.Struct(fringes=None),
+                             defects=defects,
+                             linearizer=linearizer,
+                             crosstalk=crosstalk,
+                             bfKernel=bfKernel,
+                             bfGains=bfGains,
+                             ptc=ptc,
+                             crosstalkSources=crosstalkSources,
+                             isGen3=isGen3,)
+
+        postIsr = result.exposure
+
+        if self.config.doRepairCosmics:
+            try:  # can fail due to too many CRs detected, and we always want an exposure back
+                self.log.info("Repairing cosmics...")
+                if postIsr.getPsf() is None:
+                    installPsfTask = InstallGaussianPsfTask()
+                    installPsfTask.run(postIsr)
+
+                repairConfig = CharacterizeImageTask.ConfigClass()
+                repairConfig.doMeasurePsf = False
+                repairConfig.doApCorr = False
+                repairConfig.doDeblend = False
+                repairConfig.repair.cosmicray.nCrPixelMax = 200000
+                repairTask = CharacterizeImageTask(config=repairConfig)
+
+                repairTask.repair.run(postIsr)
+            except Exception as e:
+                self.log.warn(f"During CR repair caught: {e}")
+
+        # outputExposure is persisted, exposure is returned for convenience to
+        # mimic Gen2 isrTask's API.
```

Comment:
This comment doesn't make sense: nothing is persisted in `run` here, or in the called `IsrTask`.